README.md

```diff
@@ -37,16 +37,166 @@ NOTE: we suggest using virtualenv to sandbox your setup.
+```
+
+#### Fund Senders of Contract With 0.1 Ether to Deploy TXs
+```bash
+w3.eth.sendTransaction({'to': VYPER_RLP_DECODER_TX_SENDER, 'value': 10**17})
+w3.eth.sendTransaction({'to': PURITY_CHECKER_TX_SENDER, 'value': 10**17})
+w3.eth.sendTransaction({'to': MSG_HASHER_TX_SENDER, 'value': 10**17})
+```
+
+#### Deploy Dependency Contracts
+```bash
+rlp_tx_hash = w3.eth.sendRawTransaction(VYPER_RLP_DECODER_TX_HEX)
+rlp_receipt = w3.eth.getTransactionReceipt(rlp_tx_hash)
+rlp_decoder_address = rlp_receipt.contractAddress
+
+msg_tx_hash = w3.eth.sendRawTransaction(MSG_HASHER_TX_HEX)
+msg_receipt = w3.eth.getTransactionReceipt(msg_tx_hash)
+msg_hasher_address = msg_receipt.contractAddress
+
+purity_tx_hash = w3.eth.sendRawTransaction(PURITY_CHECKER_TX_HEX)
+purity_receipt = w3.eth.getTransactionReceipt(purity_tx_hash)
+purity_checker_address = purity_receipt.contractAddress
+```
+
+### Step 5: Casper Parameters
+[Detailed Explaination of Casper Parameters](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1011.md#casper-contract-params)
+
+```bash
+EPOCH_LENGTH = 10
+WARM_UP_PERIOD = 20
+WITHDRAWAL_DELAY = 8
+DYNASTY_LOGOUT_DELAY = 5
+BASE_INTEREST_FACTOR = Decimal('0.02')
+BASE_PENALTY_FACTOR = Decimal('0.002')
+MIN_DEPOSIT_SIZE = 1000 * 10**18  # 1000 ether
+
+casper_args = [EPOCH_LENGTH, WARM_UP_PERIOD, WITHDRAWAL_DELAY, DYNASTY_LOGOUT_DELAY, msg_hasher_address, purity_checker_address, BASE_INTEREST_FACTOR, BASE_PENALTY_FACTOR, MIN_DEPOSIT_SIZE]
+```
+
+### Step 6: Compiling and Deploying Casper
+```bash
+
+file = open(os.getcwd()+('/casper/contracts/simple_casper.v.py')) #this will change depending on where the casper contract is located relative to your current working directory
+casper_code = file.read()
+
+casper_bytecode = compiler.compile(casper_code) #using vyper
+casper_abi = compiler.mk_full_signature(casper_code) #using vyper
+base_sender = base_tester.get_accounts()[-1] #account created by eth-tester containing 1 million eth
+
+
+Casper = w3.eth.contract(abi=casper_abi, bytecode=casper_bytecode) #cteates casper contract object
+tx_hash = Casper.constructor().transact({'from': base_sender}) #deploy casper contract
+tx_receipt = w3.eth.getTransactionReceipt(tx_hash)
+
+casper_address = tx_receipt.contractAddress
+
+w3.eth.sendTransaction({'to': casper_address, 'value': 10**21})
+
+casper_contract = w3.eth.contract(address=casper_address, abi=casper_abi)
+casper_contract.functions.init(*casper_args).transact()
```

Comment:
I think that it will not work with the latest version because casper_args is not correct.

Reply:
Thank you for the hint. I will double check this issue.